src/openpgp.js

```diff
@@ -32,6 +32,7 @@ import util from './util';
 
 /**
  * Generates a new OpenPGP key pair. Supports RSA and ECC keys. By default, primary and subkeys will be of same type.
+ * The primary key will not have encryption capabilities.
```

Comment:
If we're gonna write this, I think it would make sense to expand a bit and say what capabilities it does have, e.g.
```suggestion
 * By default, a primary key with signing capabilities and one subkey with encryption capabilities are generated.
```

Reply:
I think this gives the impression that you can change the capabilities of the primary key?
```suggestion
 * The generated primary key will have signing capabilities. By default, one subkey with encryption capabilities is also generated.
```

Reply:
Yeah, fair enough :+1: (Maybe we should offer the possibility to generate a primary key with only certification capabilities, and a signing subkey and an encryption subkey, but ok, that's very out of scope here :sweat_smile:)